CHANGELOG.md

```diff
@@ -17,11 +17,16 @@ the detailed section referring to by linking pull requests or issues.
 #### Added
 
 - Added login polling to prevent auto-logout when page is open.
+- Added handling of the additional properties in the case of assets
+- Added viewing of policies option in the Asset Detail Dialog
```

Comment:
Wording "option". The primary beneficiaries of the additional properties and policies are contract offers. The asset list should only contain assets you created, which mostly should not contain unknown properties.

- Now showing policies for contract offers in the details dialog.